### DISCUSSION TITLE
IK41UC LTE adapter and storage performance related questions?

### DISCUSSION THREAD

**Discussion by themajesticking** [tesla-android/issue-tracker#97] [Ideas]
Hey guys. It appears that I have everything working properly in offline mode on version 2022.38.1 but I have 2 questions. 

1. I was trying to avoid using the IK41UC LTE adapter since I have my own 5G Hotspot/Modem which has an ethernet port. Is there any way I can feed internet to the Pi4 via ethernet or USB and have it share that connection instead of the IK41UC?

2. Considering the Raspberry Pi4 supports USB boot and several people like myself probably have SSD drives laying around is to possible to get USB boot support in U-boot? I tried using one but got errors and I assumed it was a limitation of U-Boot. I would assume the use of Solid State storage might speed things up a bit. If nothing else it would help significantly with boot times.

Thanks

**Comment by themajesticking** [comment]
Disregard #1. Looks like it shares the internet connection from Ethernet perfectly!!

**Comment by mikegapinski** [comment]
2. I'll need to support booting from USB on my end, I've decided against this since it'll increase power consumption and you'll get close to saturating what the center console can output quickly when overclocking. I'll revisit when I get CPU usage down, 2033.38.1 is very bad in this regard

**Reply by themajesticking** [reply]
> 2. I'll need to support booting from USB on my end, I've decided against this since it'll increase power consumption and you'll get close to saturating what the center console can output quickly when overclocking. I'll revisit when I get CPU usage down, 2033.38.1 is very bad in this regard

I thought it was just me but my performance seems to have actually gotten worse after the overclock. This is with a large cooler/fan combo so I doubt it's a heat issue.
